libkbfs/bserver_remote.go

```diff
@@ -141,11 +248,45 @@ func (b *blockServerRemoteClientHandler) ShouldRetryOnConnect(err error) bool {
 
 var _ rpc.ConnectionHandler = (*blockServerRemoteClientHandler)(nil)
 
+func (b *blockServerRemoteClientHandler) pingOnce(ctx context.Context) {
+	_, err := b.getClient().BlockPing(ctx)
```

Comment:
Oh, this is somewhat confusing naming!  The `getClient()` here is actually an accessor function for the underlying rpc client.  `BServerRemote.getConn/putConn` is at a higher level.  So the pings do happen for both connections.